webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue

```diff
@@ -70,6 +70,7 @@ export default {
   methods: {
     save() {
       this.error = null;
+      const regex = /^[^\s]+(\s+[^\s]+)*$/;
```

Comment:
Use String.trim() and check that the text is bigger than minimum field length

Reply:
So no need to display a warning message ?

Reply:
For me yes, simply we get rid of all spaces and make the text get checked with the default rules.

Reply:
@mzorai WDYT ?

Reply:
it's OK for me to trim the seized value at the begining and the end from spaces

Reply:
If field contains more then a white space at the beginning or the end we end up in the same situation. WDYT ?